apps/web-app/src/app/features/assessment/model/assessment-response-assess-result.spec.ts

```diff
@@ -24,25 +24,47 @@ function q(
 
 describe('assessment result', () => {
```

Comment:
oznaczanie na podstawie danych to zawsze śliski temat, jako jeden z nielicznych miał testy bo... były w tym fakapy o co całkiem łatwo :)
może warto dorzucić jakiś test, że ta tablica odfiltrowanych pytań poprawnie narzuca wynik oceny obszaru?